src/LendingPool.sol

```diff
@@ -413,13 +423,9 @@ contract LendingPool is Owned, TrustedProtocol, DebtToken {
                         INTEREST RATE LOGIC
     ////////////////////////////////////////////////////////////// */
 
-    function updateInterestRate(uint64 _interestRate) external onlyOwner {
-        //Todo: Remove function after _updateInterestRate() is implemented
-        interestRate = _interestRate; //with 18 decimals precision
-    }
-
-    function _updateInterestRate() internal {
-        //ToDo
+    function updateInterestRate() external onlyOwner {
+        uint64 _utilisation = uint64(realisedDebt / totalRealisedLiquidity);
```

Comment:
First sync interest rates, otherwise realisedDebt is not up to date

Reply:
Added _syncInterests(); at start